modules/basics/src/main/resources/com/opengamma/basics/date/BusinessDayConvention.ini

```diff
@@ -0,0 +1,41 @@
+# ExtendedEnum Business Day configuration
+
+# Additional files may be added to the classpath with the same name
+# All files will be loaded, and the priority and chain fields used to determine how they are processed
+# The 'priority' value is used to select which file to process first, higher numbers have higher priority
+# The 'nextInChain' value is used to determine whether to chain on to the next lower priority file
+# The 'removedSections' value is a list of sections to remove from lower priority chained files
```

Comment:
A example of using ``removedSections`` would be useful, I'm not sure I'd know what to do from the docs